src/lib/assets/index.ts

```diff
@@ -28,6 +29,68 @@ import { watcher, moduleSecret } from "./pods";
 
 import { clusterRoleBinding, serviceAccount, storeRole, storeRoleBinding } from "./rbac";
 import { createDirectoryIfNotExists } from "../filesystemService";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function toYaml(obj: any): string {
+  return dumpYaml(obj, { noRefs: true });
+}
+
+function webhookYaml(
+  assets: Assets,
+  whc: kind.MutatingWebhookConfiguration | kind.ValidatingWebhookConfiguration,
+): string {
+  const yaml = toYaml(whc);
```

Comment:
```suggestion
  const yaml = toYaml(webhookConfiguration);
```

See earlier comment about abbreviated variable names